src/containers/FollowUnfollow/index.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { followAction, unfollowAction, fetchAction } from "./state/actions";
+
+class FollowUnfollow extends React.Component {
+  componentDidMount = () => {
+    const { fetch, username } = this.props;
+    fetch(username);
+  };
+
+  handleFollow = () => {
+    const { follow, username } = this.props;
+    follow(username);
+  };
+
+  handleUnfollow = () => {
+    const { unfollow, username } = this.props;
+    unfollow(username);
+  };
+
+  render() {
+    const { username } = this.props;
+    let isFollowing = false;
+    if (this.props && this.props.users) {
+      // eslint-disable-next-line
+      for (const user of this.props.users.following.users) {
```

Comment:
Add test for this functionality ensure that you pass `users` prop in order to trigger these lines

Reply:
Okay